newIDE/app/src/MainFrame/index.js

```diff
@@ -599,32 +591,13 @@ const MainFrame = (props: Props) => {
     [state.editorTabs]
   );
 
-  const updateWindowTitle = React.useCallback(
```

Comment:
not needed anymore?

Reply:
I've updated the ProjectTitlebar component so that it refreshes the titlebar when a prop changes (either the project name, or the file metadata or the storage provider). This avoids this needs to manually update it/store something that is redundant (and that we can forget to call). 
In other words, I removed this imperative method because we could do without it.